### ISSUE TITLE
Sync: Unexplained Behavior

### ISSUE THREAD

**Issue by shaysingh818** [shaysingh818/DTunes#26]
**Context**: The audio file module should allow you to sync files from the raw path to the database path by doing `./dtunes sync ../../data/raw/ ../../data/audio_files/`. Once files land in `data/audio_files` they should be synced to the database and ready to queue up. 

**Problem**: Even though the test cases for audio file syncing pass, executing the command sometimes doesn't sync anything. It will return a blank response and no files from raw will move. Currently it works if I print out he file names being synced but when removing the print statements, it won't do anything.
